### DISCUSSION TITLE
Toolbar menu delay tweaks

### DISCUSSION THREAD

**Discussion by Mystqr4** [WindowTop/WindowTop-App#246] [Ideas]
I have had a few problems with the Toolbar menu to pop up particularly on windows with that have media playing and more so in shrink mode. I would also like to see some sort of delay options for said menu. Some people may prefer a faster pop up whilst some people might prefer a slower one so it would be nice to have an option to tweak it by incremental numbers too.

**Comment by gileli121** [comment]
@Mystqr4 
You mean to the mouse hover time (above the toolbar arrow) before the toolbar(popup) opens right?
Currently, there is no option to configure the hover time that needed to show the toolbar/popup but it is easy to implement.
I think that I will do it.

Thanks for the feedback

**Comment by gileli121** [comment]
@Mystqr4 
It implemented as part of v5.19.2 

![image](https://user-images.githubusercontent.com/17680514/213866174-6c2bf648-88b3-4f9d-8fc9-cc7a50ac6c76.png)

It is included in the last beta release here:
https://github.com/BiGilSoft/WindowTop/issues/282

Please check this version and let me know if it works well for you, and I also want to know if the version is stable.
Thank you
